.gitignore

```diff
@@ -6,5 +6,3 @@ out/
 
 .bsp
 .scala-build
-
-*.semanticdb
```

Comment:
shouldn't we keep this line?

Reply:
I'm checking that again locally, but I think those changes make that unnecessary anymore.

Reply:
I confirm that. From clean sources, compiling everything and importing / compiling the project in / from Metals doesn't create spurious `*.semanticdb` files alongside sources.